### PR TITLE
Updated Readme.md with Content and Screenshots

### LINKED ISSUE #21: Update Readme.md file with Screenshots
@harry-hov Please assign this task to me bro

### PR DESCRIPTION
**Pull request by SAITARUN55**
Fixes #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +75,10 @@ git push -f origin master
 ```
 
 *Note: make sure that "master" is your current branch*
+
+## Getting Started
+
+For help getting started with Flutter, view our online
```
Comment (harry-hov) [inline README.md]: > For help getting started with Flutter, view our online

remove `our` or revamp the sentence

File: README.md [hunk 2]
```diff
@@ -6,6 +6,31 @@
...
 
+The app is a complete free handbook of Basics of C Programming language which covers important topics, notes & programs on the course. Download the App as a reference material for Computer Science & software Engineering programs & Degree courses.
+
+C programming is a powerful general-purpose language. If you are new to programming then C Programming is the best language to start your programming journey. In practical C programming is used in Embedded stuff, Systems Programming.
+
+## Features
+* [x] Android (iOS Coming Soon)
+
+  * [x] Contains all basic concepts of 'C' programming language (C Language).
+  * [x] C programs with comments for better understanding (300+ programs)
```
Comment (harry-hov) [inline README.md]: >C programs with comments for better understanding (300+ programs)

we do not have 300+ programs in our app right now.
maybe we will have in future.
According to me, its not right to include this.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,31 @@
...
+  * [x] Very smooth User Interface
+  * [x]
```
Comment (harry-hov) [inline README.md]: why this blank line ???

**Comment by harry-hov** [comment]
Hey @SAITARUN55,
it looks good to me. Take a look at the comments I added.
Also squash your commits into single one.

**Comment by SAITARUN55** [comment]
@harry-hov ok...bro will make the changes and one more How to squash the 4 commits to 1 single commit...could you please help me🤔

**Comment by harry-hov** [comment]
check this out https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by SAITARUN55** [comment]
i had made the necessary changes and had squashed the commits...please do look after

**Pull request merged**